Homework8/routers/api/user/index.js

```diff
@@ -0,0 +1,12 @@
+const { Router } = require("express");
+const login = require("./login");
+const register = require("./register");
+const signout = require("./signout");
+
+const router = Router();
+
+router.use("/register", register);
+router.use("/login", login);
+router.use("/signout", signout);
```

Comment:
```suggestion
router.use("/logout", signout);
```
раз у тебя `login`, то уже делай и `logout` )